internal/bminventory/inventory.go

```diff
@@ -116,7 +118,14 @@ const ignitionConfigFormat = `{
       "path": "/etc/motd",
       "mode": 644,
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-    }]
+	},
+	{
+	"filesystem": "root",
+	"path": "/etc/hosts",
+	"mode": 420,
+	"append": true,
+	"contents": { "source": "{{.ASSISTED_INSTALLER_IPS}}" }
```

Comment:
Shouldn't we only activate lines 122-128 when TARGET=onprem?

Reply:
I think yes and it will be better to have a separate config for disconnected onprem. @sacharya WDYT? Ref https://github.com/sacharya/bm-inventory/blob/onprem-disconnected/internal/bminventory/inventory.go#L124-L149

Reply:
Yes, it needs to be activated only for onprem. Disconnected is a separate usecase that we are not tackling right now.

Reply:
Done

Reply:
Currently investigating if it is possible to provide multiple ignition configs to embed to the coreos installer here https://github.com/oshercc/coreos_installation_iso/blob/master/install_process.py#L47 instead of duplicating the config for onprem case here https://github.com/pawanpinjarkar/assisted-service/blob/a8ce71fb274335f3eeb2ee945ccbda91ceeb3514/internal/bminventory/inventory.go#L123

Reply:
Added code fix to append additional configuration to the original ignition config for onprem use.